pandora-client-web/src/components/wiki/pages/items.tsx

```diff
@@ -176,20 +175,20 @@ export function WikiItems(): ReactElement {
 				So the first item is usually something like a jacket or dress, whereas underwear is further down in the list.
 			</p>
 			<p>
-				You add and remove outfits worn by the character from top to bottom, so from the outermost worn item (e.g. a jacket or dress)
+				You add and remove the items worn by the character from top to bottom, so from the outermost worn item (e.g. a jacket or dress)
```

Comment:
I'm actually not sure if "the" should be here:
```suggestion
				You add and remove items worn by the character from top to bottom, so from the outermost worn item (e.g. a jacket or dress)
```